assets/styles/components/_header.scss

```diff
@@ -233,7 +233,7 @@
     text-decoration: none;
     height: 100%;
 
-    div {
```

Comment:
it felt very brittle that the hamburger icon in the header was styled solely by being the only `<div>` element in the header. Gave it an `id` instead as there should only be one menu on the page inside the header